### PR TITLE
fix: compile error due to merge stale PR

### PR DESCRIPTION
**Pull request by xxchan**
caused by #565

### CODE REVIEW

File: crates/iceberg/src/expr/visitors/page_index_evaluator.rs [hunk 1]
```diff
@@ -24,14 +24,14 @@ use ordered_float::OrderedFloat;
...
-use parquet::format::PageLocation;
+use parquet::file::page_index::offset_index::OffsetIndexMetaData;
 
 use crate::expr::visitors::bound_predicate_visitor::{visit, BoundPredicateVisitor};
 use crate::expr::{BoundPredicate, BoundReference};
 use crate::spec::{Datum, PrimitiveLiteral, PrimitiveType, Schema};
 use crate::{Error, ErrorKind, Result};
 
-type OffsetIndex = Vec<Vec<PageLocation>>;
+type OffsetIndex = Vec<OffsetIndexMetaData>;
```
Comment (xxchan) [inline crates/iceberg/src/expr/visitors/page_index_evaluator.rs:34]: This conflicts with #626

File: crates/iceberg/src/arrow/reader.rs [hunk 2]
```diff
@@ -245,7 +245,7 @@ impl ArrowReader {
                     record_batch_stream_builder.metadata(),
                     &selected_row_groups,
                     &field_id_map,
-                    task.schema(),
+                    &task.schema,
```
Comment (xxchan) [inline crates/iceberg/src/arrow/reader.rs:248]: This conflicts with #609

**Comment by xxchan** [comment]
@Xuanwo @liurenjie1024 @sdd Should we add "require PR to be up to date" or "merge queue"? (I prefer merge queue if possible)

**Review by Xuanwo — APPROVED** [review]
Thanks!

**Pull request merged**